src/lib/components/submit/SubmitReview.svelte

```diff
@@ -30,6 +33,21 @@
 	const dispatch = createEventDispatcher();
 
 	const onSubmit = async () => {
+		if ($store?.metadata?.proposalAction === 'AddOrRemoveNodeProvider') {
```

Comment:
this can probably be solved without duplicating code given that `submitAddNodeProviderProposal` and `submitMotionProposal` accept the same parameters and provide the same results.

Can you avoid duplication?